test/IntegrationTestCommon/Factories/WebApplicationFactoryBase.cs

```diff
@@ -261,6 +240,35 @@ protected override void ConfigureWebHost(IWebHostBuilder builder)
         }
     }
 
+    private static void Replace<TService, TNewImplementation>(IServiceCollection services)
+        where TService : class
+        where TNewImplementation : class, TService
+    {
+        services.RemoveAll<TService>();
+        services.AddSingleton<TService, TNewImplementation>();
+    }
+
+    private static void Replace<TService>(IServiceCollection services, TService implementation)
+        where TService : class
+    {
+        services.RemoveAll<TService>();
+        services.AddSingleton<TService>(implementation);
+    }
+
+    public HttpClient CreateAuthedClient(string accessToken)
+    {
+        var handler = Server.CreateHandler((context) =>
+        {
+            context.Request.Headers.Authorization = $"Bearer {accessToken}";
+        });
+
+        return new HttpClient(handler)
+        {
+            BaseAddress = Server.BaseAddress,
+            Timeout = TimeSpan.FromSeconds(200),
```

Comment:
❓ Why 200?

Reply:
Stole it from here 🤷 https://github.com/dotnet/aspnetcore/blob/339c39e25bc95940f5103d978870f500463bd114/src/Hosting/TestHost/src/TestServer.cs#L180